homeassistant/components/volvooncall/__init__.py

```diff
@@ -112,6 +113,19 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up the Volvo On Call component from a ConfigEntry."""
+
+    if CONF_UNIT_SYSTEM not in entry.data.keys():
```

Comment:
Maybe add a comment when this was added, if we ever want to remove the migration code.